WalletWasabi.Gui/Config.cs

```diff
@@ -44,29 +44,21 @@ public class Config : IConfig
 		[JsonProperty(PropertyName = "UseTor")]
 		public bool? UseTor { get; internal set; }
 
-		[JsonProperty(PropertyName = "TorHost")]
-		public string TorHost { get; internal set; }
+		[JsonProperty(PropertyName = "TorSocks5EndPoint")]
+		[JsonConverter(typeof(EndPointJsonConverter), Constants.DefaultTorSocksPort)]
+		public EndPoint TorSocks5EndPoint { get; internal set; }
 
-		[JsonProperty(PropertyName = "TorSocks5Port")]
-		public int? TorSocks5Port { get; internal set; }
+		[JsonProperty(PropertyName = "MainNetBitcoinP2pEndPoint")]
+		[JsonConverter(typeof(EndPointJsonConverter), Constants.DefaultMainNetBintcoinP2pPort)]
```

Comment:
The constant name says **Bin**tcoin. The same misspelling is in other places too.

Reply:
Thanks https://github.com/zkSNACKs/WalletWasabi/pull/1945/commits/90a80f46fd2c6a9c603cdccc5c0323360cc803cd